<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="listOfProfiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveProfileButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadProfileButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteProfileButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="devices">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>832</width>
           <height>755</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="deviceListContainer"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addDeviceButton">
        <property name="text">
         <string>Add Device</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionLoad_Profile">
   <property name="text">
    <string>Load Profile</string>
   </property>
  </action>
  <action name="actionExport_Profiles">
   <property name="text">
    <string>Export Profiles</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
